setup.py

```diff
@@ -0,0 +1,42 @@
+from os import path
```

Comment:
Could you put the same license boilerplate as https://github.com/flux-framework/PerfFlowAspect/blob/main/perfflow/advice_chrome.py#L3?